### PR TITLE
add a tool to generate machine.json

### PR DESCRIPTION
**Pull request by kimeunju108**

### CODE REVIEW

File: tools/AwsClient/src/main/java/AwsClient.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            BufferedWriter bw = new BufferedWriter(fw);
+            bw.write(machineJson);
+            bw.close();
+            System.out.println("machine.json file is created.");
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+        }
+    }
+
+}
```
Comment (xieus) [inline tools/AwsClient/src/main/java/AwsClient.java]: @kimeunju108 Is it 100% necessary to include a jar file or we could generate the jar on the fly?

File: tools/AwsClient/src/main/java/AwsClient.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
+package main.java;
```
Comment (xieus) [inline tools/AwsClient/src/main/java/AwsClient.java]: I would recommend to add a README in the tools directory or tools/AwsClient directory

File: tools/AwsClient/src/main/java/AwsClient.java (hunk 1 above)
Comment (kimeunju108) [reply tools/AwsClient/src/main/java/AwsClient.java]: I removed the jar files and changed scripts.

File: tools/AwsClient/src/main/java/AwsClient.java (hunk 2 above)
Comment (kimeunju108) [reply tools/AwsClient/src/main/java/AwsClient.java]: Thank you for the recommendation. I added README.md at tools/AwsClient.

Comment (xieus) [reply tools/AwsClient/src/main/java/AwsClient.java]: Great!

**Pull request merged**